upup/pkg/fi/cloudup/awstasks/classic_load_balancer.go

```diff
@@ -80,6 +84,114 @@ func (e *ClassicLoadBalancer) CompareWithID() *string {
 	return e.Name
 }
 
+type deleteTargetGroup struct {
+	request *elbv2.DeleteTargetGroupInput
+}
+
+var _ fi.Deletion = &deleteTargetGroup{}
+
+func (d *deleteTargetGroup) TaskName() string {
+	return "TargetGroup"
+}
+
+func (d *deleteTargetGroup) Item() string {
+	return aws.StringValue(d.request.TargetGroupArn)
+}
+
+func (d *deleteTargetGroup) Delete(t fi.Target) error {
+	klog.V(2).Infof("deleting target group %v", d)
+
+	awsTarget, ok := t.(*awsup.AWSAPITarget)
+	if !ok {
+		return fmt.Errorf("unexpected target type for deletion: %T", t)
+	}
+
+	klog.V(2).Infof("Calling Nlb DeleteTargetGroup for %s", aws.StringValue(d.request.TargetGroupArn))
+
+	if _, err := awsTarget.Cloud.ELBV2().DeleteTargetGroup(d.request); err != nil {
+		return fmt.Errorf("error Deleting TargetGroup from NLB: %v", err)
```

Comment:
We should ignore "target group not found" errors here, just like we do in FindDeletions